### DISCUSSION TITLE
Monthly Usage

### DISCUSSION THREAD

**Discussion by primostasis123** [vercel/community#2248] [Help] [answered]
### Summary

Hello! As a new user of Vercel's ecosystem, I am currently using a Hobby plan to test the waters before I invest in the Pro plan. However, I was wondering why my usage hasn't been reset after a month. Am I missing something or is this how the plans work?

**Comment by amyegan** [comment]
Hi @primostasis123! I want to make sure I understand what you mean. Is it something you're seeing on your usage dashboard, or is it a limit you're running into?

**Reply by primostasis123** [reply]
yes ,  what I'm seeing in my usage dashboard.  So I was wondering why it did not reset back to zero? is it only for hooby tier?

**Reply by amyegan** [reply, marked as answer]
Yes, things work a little differently for Hobby accounts since they don't have billing cycles the same way as Pro and Enterprise teams. Instead, a rolling 30 day cycle is used. So on the Hobby account Usage page, you'll see "Last 30 Days" by default.

<img src="https://user-images.githubusercontent.com/8247105/235496071-7ddb1c49-cace-41bd-a651-bead6780c603.png" alt="Hobby usage showing last 30 days" />

And on a pro team Usage page, you'll see "Current Billing Cycle" by default.

<img src="https://user-images.githubusercontent.com/8247105/235496485-6de0f570-4792-4416-a3c8-d8ccea6e9357.png" alt="Pro usage showing current billing cycle" />
